lib/index.ts

```diff
@@ -55,6 +55,13 @@ export interface DeboredOptions {
   readonly resources?: ResourceRequirements;
 }
 
+// TODO: Need different approach if we support multiple ingress controllers in the future
+export enum IngressType {
+  SERVICE_LOADBALANCER = 'LoadBalancer',
```

Comment:
why not just `LOAD_BALANCER`?

Reply:
I think the words 'Ingress' and 'LoadBalancer' could be confusing for some users, especially people not managing infrastructures and/or clusters and just want to deploy their apps to the cluster without enough knowledge around Kubernetes.